oc_dms_mirror/dms_mirror.py

```
@@ -495,24 +495,25 @@ def _register_artifact(self, tgt_gav, ci_type):
-        params = {
-            "location": _location,
-            "citype": ci_type,
-            "depth": 0
-        }
-        message = self.psql_mq_client.compose_message('register_file', params)
-        self.logger.debug(self.__log_msg('Composed message: [%s]' % message))
-        self.psql_mq_client.enqueue_message('cdt.dlartifacts.input', message)
+        if self._args.msg_target == "amqp":
+            # Send to RabbitMQ
+            self.queue_client.connect()
+            self.logger.info(self.__log_msg(f"About to send queue to mq"))
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)